README.md

```diff
@@ -37,30 +37,13 @@ Additional configuration is required in React environments due to the removal of
 
 ## Troubleshoot
 
-If you get into issues with Buffer and polyfills check out our [fix below](https://xmtp.org/docs/developer-quickstart#troubleshooting).
+### Buffer polyfill
 
-### Create React App
+If you get into issues with Buffer and polyfills check out our [solutions](https://xmtp.org/docs/faq#why-my-app-is-failing-saying-buffer-is-not-found).
```

Comment:
```suggestion
If you run into issues with Buffer and polyfills, see this [solution](https://xmtp.org/docs/faq#why-my-app-is-failing-saying-buffer-is-not-found).
```

Reply:
BTW, we may want to update this sentence on our docs site to match. first sentence in the linked section: https://xmtp.org/docs/faq#why-my-app-is-failing-saying-buffer-is-not-found.